web/src/main/java/com/futurewei/alcor/web/entity/route/RouterUpdateInfo.java

```diff
@@ -32,6 +32,9 @@
     @JsonProperty("gateway_port_ids")
     private List<String> gatewayPortIds;
 
+    @JsonProperty("internal_router_info")
+    private InternalRouterInfo internalRouterInfo;
```

Comment:
@cj-chung @kevin-zhonghao is there always a single InternalRouterInfo for a given RouterUpdateInfo entity? Think about the possibility of an array.

Reply:
If there is related only one router, it may only have one InternalRouterInfo. I'll discuss on it with James.